angular/src/app/service/firebase/pomodoro/data.service.spec.ts

```diff
@@ -95,4 +121,66 @@ describe('DataService', () => {
     expect(testFireStore.status).toEqual(['doc', 'path', 'valueChanges', 'data']);
 
   });
+
+  // You need async here, to wait for error message
```

Comment:
Good... otherwise you'll test for console.error message before the message is sent.  Hence, you will test too soon.